modules/gdscript/gdscript_editor.cpp

```diff
@@ -1764,6 +1764,113 @@ static bool _is_expression_named_identifier(const GDScriptParser::ExpressionNode
 	return false;
 }
 
+// Creates a map of exemplary results for some functions that return a structured dictionary.
+// Setting this example as value allows autocompletion to suggest the specific keys in some cases.
+static HashMap<String, Dictionary> make_structure_samples() {
+	HashMap<String, Dictionary> res;
+	const Array arr;
+
+	{
+		Dictionary d;
```

Comment:
Not important for a fix, but keeping this list up to date won't be convenient. This feels like something that should be part of `ClassDB` and defined in the `bind_methods` near the method definition:

<img width="890" height="310" alt="Image" src="https://github.com/user-attachments/assets/6176eb2d-bfec-4872-9840-e922f09be80d" />

Reply:
I'd slap a big TODO here because these dictionaries should really be structs once implemented properly. If this list is to be inserted in **ClassDB** it should not be present in release builds as it serves no purpose.